packages/tenable_io/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.3.1"
```

```suggestion
- version: "2.4.0"
```

Since it is an enhancement and not a fix.